app/src/main/java/app/nakaura/chloe/original/LoginFragment.kt

```diff
@@ -67,7 +67,6 @@ class LoginFragment : Fragment() {
                 .addOnFailureListener { exception ->
                     Log.w(TAG, "Error getting documents.", exception)
```

Comment:
```suggestion
                    Log.d(TAG, "Error getting documents.", exception)
```
Log は `debug` の `d` に統一するのが良さそう！